bitcoin/src/psbt/map/input.rs

```diff
@@ -131,6 +133,13 @@ pub struct Input {
     /// Unknown key-value pairs for this input.
     #[cfg_attr(feature = "serde", serde(with = "crate::serde_utils::btreemap_as_seq_byte_values"))]
     pub unknown: BTreeMap<raw::Key, Vec<u8>>,
+
+    // Psbtv2 fields
+    pub previous_tx_id: Option<Txid>, // Required in PsbtV2.
```

Comment:
Make sure to have the same doccomments in the line. The same that we did for other fields.

Reply:
Same comment for other fields. Make sure that the doc-comment is the same as the text in the BIP